app/about_us/page.module.css

```diff
@@ -3,3 +3,123 @@
   width: 100vw;
   height: 100vh;
 }
+
+.crewContainer,
+.valueContainer
+ {
+  display: grid;
+  gap: 20px;
+  padding: 10px;
+}
+
+.valueItems{
+  display: flex;
+  align-items: center;
+}
+
+.value {
+  border-bottom: 1px solid rgba(88, 95, 137, 0.755);
+}
+
+.valueTitle {
+  color: gray;
+  font-size: 120px;
+  font-weight: 700;
+  padding: 20px 30px 20px 30px;
```

Comment:
When you provide only two values for padding, the first value (20px) applies to the top and bottom, and the second value (30px) applies to the left and right.

Reply:
I am going to make the changes.